docs/Deploying/FAQ.md

```diff
@@ -258,8 +258,17 @@ Check out the [documentation on running database migrations](./Upgrading-Fleet.m
 
 ## What API endpoints should I expose to the public internet?
 
-If you would like to manage hosts that can travel outside your VPN or intranet we recommend only exposing the "/api/v1/osquery" endpoint to the public internet.
+If you would like to manage hosts that can travel outside your VPN or intranet we recommend only exposing the `/api/v1/osquery` endpoint to the public internet.
 
+If you are using Fleet Desktop, we recommend exposing `/api/latest/fleet/device/*/desktop`. This minimal endpoint will only provide the number of failing policies. 
```

Comment:
Should `/api/fleet/orbit/*` and`/api/fleet/device/ping` be on that line too?

Reply:
Also, instead of saying we recommend exposing this for desktop, I would say if you want desktop to work over the internet this MUST be exposed.